src/Item.php

```diff
@@ -13,10 +13,26 @@ public function getTaxRate()
     }
 
     /**
+     * @inheritDoc
+     */
+    public function getTotalTaxAmount()
```

Comment:
This parameter is actually used in `ItemDataTrait::getItemData()`: `'total_tax_amount' => $item->getTotalTaxAmount() * 100,`